example_progs.cpp

```diff
@@ -11483,8 +11483,8 @@ prog histogram1d() {
 
 prog conv1x1() {
```

Comment:
C++ function cannot start with number, so I rename it. It runs through the frontend clockwork test.